src/coreclr/classlibnative/float/floatsingle.cpp

```diff
@@ -34,7 +35,7 @@
 FCIMPL1_V(float, COMSingle::Acos, float x)
```

Comment:
Are these still needed or can we, RyuJIT, do something like `[RuntimeImport(RuntimeLibrary, "netintrinsics_acosf")]` similar to NativeAOT

Reply:
These are still needed today.